client/reducers/index.js

```diff
@@ -4,10 +4,12 @@ import categories from './categories'
 import foodDetails from './foodDetails'
 import foods from './foods'
 import info from './info'
+import sort from './sort'
 
 export default combineReducers({
   categories,
   foodDetails,
   foods,
-  info
+  info,
+  sort
```

Comment:
Sorry - could you follow the positioning in the [wiki](https://github.com/matai-2019/foodlist/wiki/Redux-store).